cypress/support/api_commands.js

```diff
@@ -397,13 +395,3 @@ Cypress.Commands.add('apiGetConfig', () => {
     // # Get current settings
     return cy.request('/api/v4/config');
 });
-
-// *****************************************************************************
-// Post creation
-// *****************************************************************************
-
-Cypress.Commands.add('postMessageAs', (sender, message, channelId, createAt = 0) => {
-    cy.task('postMessageAs', {sender, message, channelId, createAt}).
-        its('status').
-        should('be.equal', 201);
-});
```

Comment:
I removed this due to some Cypress priority when running a `task`; needed to find a way to better handle getting `baseUrl`.  Besides, I found this confusing to include in `api_command`.  I'm hoping to limit adding command in this file only to those with direct implementation of https://api.mattermost.com/.

Reply:
This is fine, I think we need to refactor our custom commands a bit. Added a note for our automation meeting to discuss.